tests/test_playbooks/vars/server.yml.example

```diff
@@ -5,8 +5,10 @@ foreman_password: "changeme"
 foreman_server_url: "https://foreman.example.com"
 foreman_validate_certs: false
 
+foreman_proxy: "https://foreman.example.com"
```

Comment:
is it correct to have it *with* the `https://` schema prefix?

Reply:
I guess not...